package.json

```diff
@@ -48,7 +48,7 @@
 		"@datadog/datadog-api-client": "^1.0.0-beta.7",
 		"@hashicorp/platform-cli": "^2.8.0",
 		"@hashicorp/platform-configs": "^0.2.0",
-		"@hashicorp/platform-nextjs-plugin": "^5.1.0",
+		"@hashicorp/platform-nextjs-plugin": "^5.1.1",
```

Comment:
Is this change related to the blurry SVG issue? Or is it separate? 

Also, I'm noticing that [rolling back this version](https://github.com/hashicorp/dev-portal/pull/2510) seems to result in the same bundle size.

Reply:
Yes when I used the inline svg for the images, the svg on the hcp page was cut off. That was because the product icon svg had a clip-path id of a and this svg had a clip-path id of a as well. I had to adjust our svg loader in the nextjs-plugin to address this. There is more info in the PR: https://github.com/hashicorp/web-platform-packages/pull/268

Also for bundle size, I assumed it was from this addition but it looks like it was not. I'll take a look at the bundle size today and see if I can decrease it